nodebuilder/core/tls.go

```diff
@@ -0,0 +1,45 @@
+package core
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/celestiaorg/celestia-node/libs/utils"
+)
+
+const xtoken = "xtoken.json"
+
+func EmptyTLSConfig() *tls.Config {
+	return &tls.Config{MinVersion: tls.VersionTLS12}
+}
+
+type AuthToken struct {
```

Comment:
we don't need this struct really, let's make it local to the function and just return string. No need for it to be separate from func or even exported.

Reply:
Makes sense. Thank you